CHANGELOG.rst

```diff
@@ -18,6 +18,81 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 3.8.0 (2018-09-05)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- `#2452 <https://github.com/pytest-dev/pytest/issues/2452>`_: ``Config.warn`` has been deprecated, it should be replaced by calls to the standard ``warnings.warn``.
+
+  ``Node.warn`` now supports two signatures:
+
+  * ``node.warn(PytestWarning("some message"))``: is now the recommended way to call this function. The warning
+    instance must be a ``PytestWarning`` or subclass  instance.
+
+  * ``node.warn("CI", "some message")``: this code/message form is now deprecated and should be converted to
+    the warning instance form above.
+
+  ``RemovedInPytest4Warning`` and ``PytestExperimentalApiWarning`` are now part of the public API and should be accessed
+  using ``pytest.RemovedInPytest4Warning`` and ``pytest.PytestExperimentalApiWarning``.
+
+
```

Comment:
we might want to consider splitting deprecations and removals

Reply:
Hmm good point